src/node-jspm-jasmine.js

```diff
@@ -134,12 +142,14 @@ export function runTests(opts, errCallback = function() {}) {
 					const fileKey = getFileKey(getOSFilePath(load.address), process.cwd());
 					// exclude the dependency modules (i.e. libraries) from instrumentation
 					if (coverageFiles[fileKey]) {
+						mkdirp.sync(tempDirectory + fileKey.substring(0, fileKey.lastIndexOf('/')));
 						// put file's transpiled counterpart in temp folder
 						let filename;
 						let sourceMap = '';
+
 						// arrange sourcemaps
 						if (load.metadata.sourceMap) {
-							filename = path.join(tempDirectory, fileKey.replace(/\//g, '|'));
+							filename = path.join(tempDirectory, fileKey);
```

Comment:
@joeldenning I would only do that if I had some strong reasons. Aside for it being more work for no real benefit (excluding aesthetics), I opted for flat because that `tmp` directory is a few directories deep already (and could be deeper depending where the user has the `node_modules` folder). Windows doesn't process very deep directories. But risk is not THAT high, so if you feel better having them be actual directories, then I say go for it until someone complains.